src/components/dropdown-menu/DropdownMenu.scss

```diff
@@ -17,6 +17,33 @@
 
     z-index: $menu-z-index;
     margin-top: 5px;
-    perspective: 500px;
-    perspective-origin: center top;
```

Comment:
Removed this because it is unnecessary and conflicts with the responsive styling